src/plot.cpp

```diff
@@ -1195,9 +1202,46 @@ void ProjectionPlot::create_output() const
   double p0 = static_cast<double>(pixels_[0]);
   double p1 = static_cast<double>(pixels_[1]);
   double vert_fov_radians = horiz_fov_radians * p1 / p0;
-  double dphi = horiz_fov_radians / p0;
-  double dmu = vert_fov_radians / p1;
 
+  // This can be changed to alter the perspective distortion effect.
+  // This is in units of cm. Most of the time does not need to be changed.
+  constexpr double focal_plane_dist = 10.0;
```

Comment:
"most of the time..." is jumping out at me here. Can we put a TODO here to make sure we remember that it may be necessary to expose it as a parameter in the future?

Reply:
done!

Reply:
:grimace: Mind making this an issue? I know I'll forget about it otherwise.